code/modules/atmospherics/machinery/portable/canister.dm

```diff
@@ -62,11 +62,7 @@ GLOBAL_LIST_INIT(gas_id_to_canister, init_gas_id_to_canister())
 	var/can_max_release_pressure = (ONE_ATMOSPHERE * 10)
 	///Minimum pressure allower for release_pressure var
 	var/can_min_release_pressure = (ONE_ATMOSPHERE * 0.1)
-	///Max amount of heat allowed inside of the canister before it starts to melt (different tiers have different limits)
-	var/heat_limit = 10000
```

Comment:
Is this supposed to be that low?

Reply:
Yes, after talking with potato on discord we came to the conclusion that the original was too high for the shielding to have uses and we wanted it to be used as often as possible

Reply:
Suffering